src/pages/api/auth/login.ts

```diff
@@ -0,0 +1,81 @@
+import {NextApiRequest, NextApiResponse} from 'next';
+import Cookies from 'cookies';
+import {withIronSessionApiRoute} from 'iron-session/next';
+import {gql} from '@apollo/client';
+import {ironSessionOptions} from '@/lib/utils/iron-session';
+import client, {setClientToken} from '@/apollo-client';
+
+export default withIronSessionApiRoute(handler, ironSessionOptions);
+
+async function handler(req: NextApiRequest, res: NextApiResponse<any>) {
+  if (req.method === 'POST') {
+    const {email, password} = req.body;
+    if (!email || !password) {
+      res.status(400);
+      res.json({error: 'Missing credentials'});
```

Comment:
I think it could be a good idea to have a file full of constants such as `Missing credentials`, and the import them from said file. This will make our responses more consistent.